SpatialGDK/Documentation/content/glossary.md

```diff
@@ -282,7 +282,7 @@ SpatialOS uses the schema to generate code. You can use this generated code in y
 > * [Reference: schema]({{urlRoot}}/content/how-to-use-schema)
 
 ### SpatialOS command-line tool (CLI)
-The SpatialOS command-line tool (also known as the “CLI”) provides a set of commands that you use to interact with a [SpatialOS project](https://docs.improbable.io/reference/latest/shared/reference/project-structure#structure-of-a-spatialos-project). Among other things, you use it to [deploy](#deployment) your game (using [`spatial local launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch) or [`spatial cloud launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-cloud-launch)). You can run the CLI commands `spatial build` and `spatial local launch` from the [GDK toolbar]({{urlRoot}}/content/toolbars#spatialos-gdk-for-unreal-toolbar) in the Unreal Editor.
+The SpatialOS command-line tool (also known as the “CLI”) provides a set of commands that you use to interact with a [SpatialOS project](https://docs.improbable.io/reference/latest/shared/reference/project-structure#structure-of-a-spatialos-project). Among other things, you use it to [deploy](#deployment) your game (using [`spatial local launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch) or [`spatial cloud launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-cloud-launch)). You can run the CLI commands `spatial build` and `spatial local launch` from the [GDK toolbar]({{urlRoot}}/content/unreal-editor-interface/toolbars#spatialos-gdk-for-unreal-toolbar) in the Unreal Editor.
```

Comment:
This might no longer be accurate after the Fast Local Restart changes, @joshuahuburn to confirm.

Reply:
`You can run the CLI commands `spatial build`` is definitely incorrect. You can't run spatial build from the toolbar and we never support `spatial build` from anywhere.
And correct, we no longer use `spatial local launch` in the toolbar but an equivalent `spot alpha deployment create`. It's probably just not worth mentioning the toolbar at all in this section. You can still run `spatial local launch` from the command line if you'd like to spawn a deployment in the classic way.